docs/configuration/sinks/rocketchat.rst

```diff
@@ -0,0 +1,63 @@
+Rocket.Chat
+#################
+
+Robusta can report issues and events in your Kubernetes cluster to Rocket.Chat.
+
+.. image:: /images/rocketchat1.png
```

Comment:
The image is taking too much screen space. Reduced height and a working graph should work better